pom.xml

```diff
@@ -5,20 +5,20 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.eclipse.gef_root</groupId>
 	<artifactId>org.eclipse.gef.releng</artifactId>
-	<version>3.19.0-SNAPSHOT</version>
+	<version>3.20.0-SNAPSHOT</version>
 	<packaging>pom</packaging>
 
 	<!-- this is the parent POM from which all modules inherit common settings -->
 	<properties>
 		<maven-antrun-plugin.version>1.7</maven-antrun-plugin.version>
 		<maven.compiler.source>17</maven.compiler.source>
 		<maven.compiler.target>17</maven.compiler.target>		
-		<tycho-version>4.0.4</tycho-version>
+		<tycho-version>4.0.6</tycho-version>

```

Comment:
@ptziegler I was not sure if this line could be removed as we have the tycho version anyhow in the maven.config

Reply:
I think so. I'm not sure whether m2e is able to process the maven.config file but a quite test doesn't produce any workspace errors.
This would only be relevant for the documentation, as they are the only plugins using the Maven nature.

And in that case, I would even go one step further and also remove the `tycho.version` as well, and replace all references to it with `tycho-version`.

Reply:
removed it lets see what happens.